src/jarabe/model/shell.py

```diff
@@ -668,8 +668,7 @@ def _active_window_changed_cb(self, screen, previous_window=None):
                 window = window.get_transient()
 
         act = self._get_activity_by_xid(window.get_xid())
-        if act is not None:
-            self._set_active_activity(act)
+        self._set_active_activity(act)
```

Comment:
You know @samdroid-apps even without https://github.com/sugarlabs/sugar/commit/e1ae8d6fc79755f3770ff30e6f50215420da185f the journal button in the home view is broken for me.